tools/generate_report.rb

```diff
@@ -0,0 +1,35 @@
+include ActionDispatch::Routing
+include Rails.application.routes.url_helpers
+
+USER_ID       = "admin".freeze
+REPORT_PARAMS = {:userid => USER_ID, :mode => "async", :report_source => "Requested by user"}.freeze
+
+def report_from_args
+  if ARGV.empty?
+    MiqReport.last
```

Comment:
it is because when I am working on report then I am usually creating copy of report(custom report) and then it is last. so then I don't need enter name or ID. so this is the reason why I think that it is helpful.